### DISCUSSION TITLE
Fast music fetching

### DISCUSSION THREAD

**Discussion by imPratyoosh** [HemantKArya/BloomeeTunes#93] [Ideas]
Hey, Bro I've tried several open-source YouTube Music clients and found that METROLIST is one of the fastest for retrieving song data and streaming songs instantly. I wanted to know if it's possible to achieve the same instant streaming experience, using the same approach METROLIST uses.

**Comment by HemantKArya** [comment]
Thanx you @PRATYOOSH-69 😊,
Try out the latest update maybe you will be surprised ;)

**Reply by imPratyoosh** [reply]
"Thanks for the update! I’ve tried the latest version, and I can see some noticeable improvements. The app feels more responsive, and the streaming experience is smoother. It’s a great step forward! Looking forward to more updates and features in the future. Keep up the good work!"

**Comment by BVISHNU78** [comment]
Music fetch issue if any song playing from an album there is no  feature view album or take another song from same album need to search and music fetching is worst some time it play same song or stop playing error message fetch media error jiosaavan playing also issue most songs are from YouTube not from jiosaavan playlist fetch from spotify 60 to 80% songs not playable.
